awx/main/redact.py

```diff
@@ -13,11 +11,13 @@ class UriCleaner(object):
     @staticmethod
     def remove_sensitive(cleartext):
         # exclude_list contains the items that will _not_ be redacted
-        exclude_list = [settings.PUBLIC_GALAXY_SERVER['url']]
-        if settings.PRIMARY_GALAXY_URL:
-            exclude_list += [settings.PRIMARY_GALAXY_URL]
-        if settings.FALLBACK_GALAXY_SERVERS:
-            exclude_list += [server['url'] for server in settings.FALLBACK_GALAXY_SERVERS]
+        # TODO: replace this with the server URLs from proj.org.credentials
```

Comment:
I still need to refactor this.

Reply:
Actually, @AlanCoding I don't think we actually need this anymore at all - the point of this code is to filter HTTP basic auth creds out of URLs - starting w/ this PR we're *only* supporting tokens, so I don't actually think we need this anymore.